.ci.yaml

```diff
@@ -3231,7 +3231,18 @@ targets:
     properties:
       tags: >
         ["devicelab","ios32","mac"]
-      task_name: native_ui_tests_ios32
+      task_name: native_ui_tests_ios
+    scheduler: luci
+
+  - name: Mac_ios native_ui_tests_ios
+    recipe: devicelab/devicelab_drone
+    bringup: true
+    presubmit: false
+    timeout: 90
```

Comment:
```suggestion
    timeout: 60
```

Reply:
The existing `Mac_ios32 native_ui_tests_ios32` one was 90, should they both be 60?

Reply:
The timeout difference is based on the platform they're running on. ios32 is 90 minutes, but default devicelab timeout is 60

Reply:
Gotcha, thanks!

Reply:
Would it be a reasonable feature request to allow us to leave off the `timeout` to use the default based on where it's running? If so I can file that...

Reply:
That's a good feature request, please file it :)

Right now, we support a default of 30 minutes, but it can't be set in the platform_properties

Reply:
https://github.com/flutter/flutter/issues/89690